src/Grpc.AspNetCore.Server/Internal/HttpContextServerCallContext.cs

```diff
@@ -275,6 +374,11 @@ public void Initialize()
             HttpContext.Response.Headers.Append(GrpcProtocolConstants.MessageEncodingHeader, ResponseGrpcEncoding);
         }
 
+        private void RequestAborted()
+        {
+            _deadlineCts?.Cancel();
```

Comment:
Not much point in calling Cancel on the deadline cts if the request was already aborted. As long as Dispose gets called eventually.

Reply:
It is canceled on abort so the token on ServerCallContext is raised both for aborts and deadlines

https://github.com/grpc/grpc-dotnet/blob/f721fbef75e485b252588002da4c7b919666f9c8/src/Grpc.AspNetCore.Server/Internal/HttpContextServerCallContext.cs#L202